### DISCUSSION TITLE
Accessing event arguments?

### DISCUSSION THREAD

**Discussion by worldpeace-germany** [missionpinball/missionpinball#116] [Q&A]
Hello,

let's  take a look at this event (as an example)

https://missionpinball.org/events/machine_var_machine_var/

It has an argument "value". Now the question is how can I access that value when that event is posted by mpf? For example I would like to display that argument "value" on a slide. So having that argument copied over to a custom variable would be one solution or maybe I can directly use it in a slide_player.

Anyone an idea?

Thanks!

**Comment by atummons** [comment]
Here is how to access variables: https://missionpinball.org/config/instructions/dynamic_values/You can see here about using the values dynamically here: https://missionpinball.org/config_players/event_player/#dynamic-event-argumentsSent from my iPhoneOn Oct 21, 2023, at 1:01 AM, worldpeace-germany ***@***.***> wrote:﻿
Hello,
let's  take a look at this event (as an example)
https://missionpinball.org/events/machine_var_machine_var/
It has an argument "value". Now the question is how can I access that value when that event is posted by mpf? For example I would like to display that argument "value" on a slide. So having that argument copied over to a custom variable would be one solution or maybe I can directly use it in a slide_player.
Anyone an idea?
Thanks!

**Comment by worldpeace-germany** [comment]
That is not what I want - or I don't see how. The links provided show what I can do with variables if a certain event occurs and could do that even conditionally only if a certain variable fulfills a certain condition. Let me try to rephrase what I want:

Example player_added event https://missionpinball.org/events/player_added/.

I can catch that event using an event_player and then I can increase a variable by a certain value, that is fine. But this event, you can see either in the log or above documentation link or in the code

        self.machine.events.post('player_added',
                                 player=player,
                                 num=player.number,
                                 callback=self._player_added)

that it has a parameter num. Now in my event player when the event player_added is being caught I do not want increase a certain variable by a fixed value but I want to increase that variable by the value posted in num. Can I do that?
